private-templates-service/src/controllers/template.ts

```diff
@@ -1,19 +1,78 @@
 import passport from '../config/passport';
 import express from 'express';
+import { Request, Response, NextFunction } from 'express';
+import { Template } from '../models/Template';
+import mongoose from 'mongoose';
+import logger from '../util/logger';
+import jws from 'jws';
+import { check, validationResult } from 'express-validator';
 
 const templateRouter = express.Router();
 
 templateRouter.all('/', passport.authenticate('oauth-bearer', {
     session: false
-    }), function(req: any, res: any, next: Function){
+    }), function(req: Request, res: Response, next: NextFunction){
     next();
 });
 
-// create a GET route
-templateRouter.get('/', (req : any, res : any) => {
-    res.send({
-        express: "YOUR EXPRESS BACKEND IS CONNECTED TO REACT. CONFIRMED"
+// GET templates route
+const getTemplates = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
Convention is to prefix unused variables with '_' -` _req`

Reply:
Fixed, thanks!